SPECS/igc-module.spec

```
@@ -0,0 +1,54 @@
+%define uname  %{kernel_version}
+%define module_dir updates
+
+Summary: Driver for igc-module
+Name: igc-module
+Version: 4.20.17
+Release: %{?release}%{!?release:1}
+License: GPL
+Source: %{name}-%{version}.tar.gz
+
+BuildRequires: kernel-devel
+Provides: vendor-driver
+Requires: kernel-uname-r = %{kernel_version}
+Requires(post): /usr/sbin/depmod
+Requires(postun): /usr/sbin/depmod
+
+%description
+igc-module Linux Device Driver source.
+
+%prep
+%setup -q -n %{name}-%{version}
```

```suggestion
%autosetup -n %{driver_name}-%{version}
```

Let's be future proof. There may be patches in the future.